utils_linux.go

```diff
@@ -171,6 +171,9 @@ func createPidFile(path string, process *libcontainer.Process) error {
 	if err != nil {
 		return err
 	}
+	if _, err := os.Stat(path); err == nil {
+		return fmt.Errorf("file %s already exists", path)
```

Comment:
I'm not sure this way of solving the issue makes sense. There might be cases where you want to overwrite a stale pidfile. Also, this should be `Lstat` AFAICS (if `path` is a broken symlink this check won't be hit).